packages/frontend/src/utils/getWalletURL.js

```diff
@@ -36,8 +24,9 @@ export const getMyNearWalletUrlFromNEARORG = (https = true) => {
     return `${https ? 'https://' : ''}${prefix || ''}mynearwallet.com`;
 };
 
+// TODO: remove?
 export const getMeteorWalletUrl = () => {
```

Comment:
Double-check it and need to decide what to do

Should we delete migration stuff?

Reply:
I guess we should let it be there until transition is finished.
They may be useful In case we will need to update something in those migration files (find a bug or smth)

Reply:
👌